op-node/rollup/sequencing/sequencer_chaos_test.go

```diff
@@ -253,20 +276,56 @@ func TestSequencerChaos(t *testing.T) {
 func testSequencerChaosWithSeed(t *testing.T, seed int64) {
 	// Lower the log level to inspect the mocked errors and event-traces.
 	logger := testlog.Logger(t, log.LevelCrit)
-	seq, deps := createSequencer(logger)
+
+	// Create test dependencies
+	rng := rand.New(rand.NewSource(123))
```

Comment:
Should this be 
```suggestion
	rng := rand.New(rand.NewSource(seed))
```
?
`rng` is initialized again at further control flow.